libs/plugin-elliptic/tests/ed25519.spec.ts

```diff
@@ -19,22 +20,22 @@ describe('ed25519 - EDDSA', () => {
   const EC = require('elliptic');
   var eddsa = EC.eddsa;
   const ed25519 = new eddsa('ed25519');
-  const messageReference= 'eyJhbGciOiJFZERTQSJ9.RXhhbXBsZSBvZiBFZDI1NTE5IHNpZ25pbmc';
+  const messageReference = 'eyJhbGciOiJFZERTQSJ9.RXhhbXBsZSBvZiBFZDI1NTE5IHNpZ25pbmc';
   const message = Buffer.from(messageReference);
-  const secretReference = [0x9d ,0x61 ,0xb1 ,0x9d ,0xef ,0xfd ,0x5a ,0x60 ,0xba ,0x84 ,0x4a ,0xf4 ,0x92 ,0xec ,0x2c ,0xc4 ,0x44 ,0x49 ,0xc5 ,0x69 ,0x7b ,0x32 ,0x69 ,0x19 ,0x70 ,0x3b ,0xac ,0x03 ,0x1c ,0xae ,0x7f ,0x60];
-  const publicReference = [0xd7 ,0x5a ,0x98 ,0x01 ,0x82 ,0xb1 ,0x0a ,0xb7 ,0xd5 ,0x4b ,0xfe ,0xd3 ,0xc9 ,0x64 ,0x07 ,0x3a ,0x0e ,0xe1 ,0x72 ,0xf3 ,0xda ,0xa6 ,0x23 ,0x25 ,0xaf ,0x02 ,0x1a ,0x68 ,0xf7 ,0x07 ,0x51 ,0x1a];
-  const signatureReference= '860c98d2297f3060a33f42739672d61b53cf3adefed3d3c672f320dc021b411e9d59b8628dc351e248b88b29468e0e41855b0fb7d83bb15be902bfccb8cd0a02'.toUpperCase();
+  const secretReference = [0x9d, 0x61, 0xb1, 0x9d, 0xef, 0xfd, 0x5a, 0x60, 0xba, 0x84, 0x4a, 0xf4, 0x92, 0xec, 0x2c, 0xc4, 0x44, 0x49, 0xc5, 0x69, 0x7b, 0x32, 0x69, 0x19, 0x70, 0x3b, 0xac, 0x03, 0x1c, 0xae, 0x7f, 0x60];
```

Comment:
consider using JSON  + base64url / hex, and a static fixture file instead of doing this.

Reply:
if these values are pulled from RFCs / test vectors a comment should be used to explain where they came from.